packages/amplify-category-auth/resources/auth-custom-resource/hostedUIProviderLambda.js

```diff
@@ -1,27 +1,33 @@
 const response = require('cfn-response');
 const aws = require('aws-sdk');
+
 const identity = new aws.CognitoIdentityServiceProvider();
 
-exports.handler = (event, context, callback) => {
+exports.handler = (event, context) => {
   try {
     const userPoolId = event.ResourceProperties.userPoolId;
-    let hostedUIProviderMeta = JSON.parse(event.ResourceProperties.hostedUIProviderMeta);
+    const hostedUIProviderMeta = JSON.parse(event.ResourceProperties.hostedUIProviderMeta);
 
-    let deleteIdentityProvider = (providerName) => {
-      let params = { ProviderName: providerName, UserPoolId: userPoolId };
+    const deleteIdentityProvider = (providerName) => {
+      const params = { ProviderName: providerName, UserPoolId: userPoolId };
       return identity.deleteIdentityProvider(params).promise();
     };
 
-    let providerPromises = [];
+    const providerPromises = [];
 
     hostedUIProviderMeta.forEach(({ ProviderName }) => providerPromises.push(deleteIdentityProvider(ProviderName)));
 
     Promise.all(providerPromises)
       .then(() => {
-        response.send(event, context, response.SUCCESS, {});
+        response.send(event, context, response.SUCCESS);
       })
       .catch((err) => {
         console.log(err.stack);
+
+        if (err.name === 'NotFoundException') {
+          return response.send(event, context, response.SUCCESS);
```

Comment:
keep in mind that `response.send` doen't return anything. See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-lambda-function-code-cfnresponsemodule.html .
I'd move return to separate line below or just add `else`

Reply:
I `return`'d it to keep it in the same line, but I can move it so that its clear I'm not returning anything